src/sentry/static/sentry/app/api.jsx

```diff
@@ -192,11 +192,11 @@ export class Client {
     let path = '/issues/' + params.id + '/';
     let id = this.uniqueId();
 
-    GroupActions.assignTo(id, params.id, {email: params.email});
+    GroupActions.assignTo(id, params.id, {email: params.member && params.member.email || ''});
 
     return this._wrapRequest(path, {
       method: 'PUT',
-      data: {assignedTo: params.email},
+      data: {assignedTo: params.member && params.member.id || 0},
```

Comment:
To be fair, it was more unclear before since it implicitly sent an empty string. :) But yeah, will add a comment.